<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="xinhuo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>科大讯飞_星火模型</string>
   </property>
   <widget class="QLineEdit" name="appid_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>APPID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>APISecret</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>APIKey</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="api_secret_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="api_key_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>星火Model</string>
    </property>
   </widget>
   <widget class="QComboBox" name="domain_combobox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>261</width>
      <height>36</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>general</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>generalv2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>generalv3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>generalv3.5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="spark_url_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="wenxin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>百度_文心一言</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>API Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Secret Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wx_appid_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="wx_api_secret_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qianwen">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>阿里_通义千问</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>API-KEY</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qw_appid_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>128</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>保存|退出</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/snail/qrc/link_2.png</normaloff>:/snail/qrc/link_2.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>所有参数在对应的网站注册以后可获取，获取方法，网站有详细介绍。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>   根据使用中的 GPT大模型 设置 参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="qxjyclass.qrc">:/snail/qrc/64snailgpt.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qxjyclass.qrc"/>
 </resources>
 <connections/>
</ui>
